src/rules/converters/codelyzer/template-cyclomatic-complexity.ts

```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../converter";
+
+export const convertTemplateCyclomaticComplexity: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "@angular-eslint/template/cyclomatic-complexity",
+            },
```

Comment:
This rulename should map the input rule arguments into the `maxComplexity` object property. Take a look at [component-class-suffix](https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/src/rules/converters/codelyzer/component-class-suffix.ts) rule converter to see how it is done.

Reply:
Also, I am not sure if the `ruleName` has the proper value as it seems nested inside `eslint-plugin-template` directory. [ESLint cyclomatic-complexity](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/rules/cyclomatic-complexity.ts)

Reply:
Changes made; tslint rule has example `[true, 4]` angular-eslint expects `{ maxComplexity: 4 }` and it appears `tslintRule.ruleArguments` skips the `true`.